client/post-editor/editor-gutenberg-opt-in-notice/style.scss

```diff
@@ -0,0 +1,53 @@
+/** @format */
+
+.editor-gutenberg-opt-in-notice {
+	margin: 0 auto;
+	width: auto;
+	position: fixed;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	opacity: 0;
+	animation: slide-in-bottom cubic-bezier( 0.175, 0.885, 0.32, 1.275 ) 0.3s forwards;
+	animation-delay: 4s;
+	box-shadow: 0 2px 6px rgba( 0, 0, 0, 0.25 );
+	z-index: z-index( 'root', '.notices-list.is-pinned' );
```

Comment:
Instructions at https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#z-index for what's happening here.

Our z-index mapper is more complex, but we also have many stacking contexts. I personally avoid use of z-index when possible in favor of natural stacking orders.

We shouldn't reuse this one ;) I won't block on this, but we can tidy this up to:
```scss
z-index: z-index( 'root', '.editor-gutenberg-opt-in-notice;' );
```

With a corresponding entry in _z-index.scss:
```scss
z-layers: (
	'root': (
//...
'.editor-gutenberg-opt-in-notice': 180,
//...
```

<img width="928" alt="screen shot 2018-10-30 at 2 57 40 pm" src="https://user-images.githubusercontent.com/1270189/47753273-32a77900-dc54-11e8-8b38-581cfa43a3ba.png">

Also to figure out a parent stacking context we can use:
https://github.com/gwwar/z-context

Reply:
Whoa that is some very cool black magic. ✨ Thanks @gwwar! Added a new stacking context and verified it's still looking good.